BoxContentSDK/BoxContentSDK/Requests/BOXFolderPaginatedItemsRequest.m

```diff
@@ -60,10 +60,36 @@ - (BOXAPIOperation *)createOperation
 
 - (void)performRequestWithCompletion:(BOXItemArrayCompletionBlock)completionBlock
 {
+    [self performRequestWithCached:nil refreshed:completionBlock];
+}
+
+- (void)performRequestWithCached:(BOXItemArrayCompletionBlock)cacheBlock refreshed:(BOXItemArrayCompletionBlock)refreshBlock
+{
     BOOL isMainThread = [NSThread isMainThread];
     BOXAPIJSONOperation *folderOperation = (BOXAPIJSONOperation *)self.operation;
-
-    if (completionBlock) {
+    
+    if (cacheBlock) {
+        if (self.requestCache) {
```

Comment:
Curious: can cacheBlock and refreshBlock exist independently of each other? e.g. refreshBlock is non-nil, cacheBlock is nil.